### PR TITLE
feature: update homepage and title (#83)

### PR DESCRIPTION
**Pull request by rohitdasu**
### 👨‍💻 Changes proposed

Updated homepage and app title

### 📷 Screenshots

Screenshot 2022-08-15 at 4 04 52 PM
Screenshot 2022-08-15 at 4 04 26 PM

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,31 +1,39 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
 import Image from 'next/image';
+import Link from 'next/link';
 
 const Home: NextPage = () => {
   return (
-    <div className='flex min-h-screen flex-col items-center justify-center py-2'>
+    <div className='min-h-screenpy-2'>
```
Comment (subhoghoshX) [inline pages/index.tsx:8]: There should be a space between `min-h-screen` and `py-2`. Or remove them if they're not necessary.

File: pages/index.tsx [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+            <Image src='/github.svg' className='cursor-pointer' height={30} width={30} alt='github' />
+          </Link>
+          <Link href='https://discord.gg/aJp6Fypb' rel='noreferrer'>
+            <Image src='/discord.svg' className='cursor-pointer' height={30} width={30} alt='discord' />
+          </Link>
+          <Link href='https://twitter.com/ykdojo' rel='noreferrer'>
+            <Image src='/twitter.svg' className='cursor-pointer' height={30} width={30} alt='twitter' />
+          </Link>
+        </div>
+      </div>
```
Comment (subhoghoshX) [inline pages/index.tsx:25]: Just nitpicking. You could make the container div a `header` and the div wrapping the links a `nav` element.

Comment (subhoghoshX) [reply pages/index.tsx:25]: the div wrapping `defaang` could also be a link pointing to `/`

File: pages/index.tsx [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
+      <main className='flex h-[calc(100vh-4rem)] w-full flex-col items-center justify-center px-[5%]  text-center lg:h-[calc(100vh-6rem)] lg:px-[20%]'>
+        <h3 className='text-3xl font-bold text-gray-800 lg:text-5xl'>Welcome to defaang 🎉</h3>
```
Comment (subhoghoshX) [inline pages/index.tsx:27]: An `h1` would be more semantic here :smiley:

**Review by subhoghoshX — COMMENTED** [review]
Overall LGTM. One last thing, try to avoid using arbitrary values as much as possible.

File: pages/index.tsx (hunk 2 above)
Comment (rohitdasu) [reply pages/index.tsx:25]: i'm thinking of making it as a separate component, what would you say?

File: pages/index.tsx (hunk 1 above)
Comment (rohitdasu) [reply pages/index.tsx:8]: ok

File: pages/index.tsx (hunk 3 above)
Comment (rohitdasu) [reply pages/index.tsx:27]: ok

**Comment by subhoghoshX** [comment]
Another thing, you need to add a link (`a` tag) within the next.js `Link` components. Please, refer to the documentation.

**Review by subhoghoshX — APPROVED** [review]

**Pull request merged**